webpack-blocks.js

```diff
@@ -0,0 +1,88 @@
+const { entryPoint, group, setOutput, sourceMaps } = require('@webpack-blocks/webpack2');
```

Comment:
Webpack blocks massively simplified the config for me and allowed use of preset style stuff. Also, there wasn't a need to change configs from Wp1 to Wp2, so win. I know what you mean, though.